pkgs/development/python-modules/rsyscall/default.nix

```diff
@@ -0,0 +1,70 @@
+{ lib
+, pythonPackages
+, pythonAtLeast
+, fetchPypi
+, librsyscall
+, nix
+, socat
+, pkg-config
+, openssh
+, coreutils
+}:
+
+pythonPackages.buildPythonPackage rec {
+  pname = "rsyscall";
+  version = "0.0.3";
+  disabled = ! pythonAtLeast "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1wvviglibnkkh332f27hqzjyz7ipbp6h3p845h0x77y1vlsf4gxk";
+  };
+
+  buildInputs = [
+    pythonPackages.cffi
+    librsyscall
+  ];
+  nativeBuildInputs = [
+    pkg-config
+  ];
+  propagatedBuildInputs = with pythonPackages; [
+    trio
+    typeguard
+    pyroute2
+    outcome
+    nixdeps
+  ];
+  # rsyscall uses these to build containers on the fly
+  exportReferencesGraph = [
+    "nix" nix
+    "librsyscall" librsyscall
+    "openssh" openssh
+    "coreutils" coreutils
+  ];
+
+  checkInputs = with pythonPackages; [
+    pytest
+    pydocstyle
+    mypy
```

Comment:
```suggestion
```
we don't need to lint the code. We want to run tests which make sure it actually works.